src/ContractInteract/Anchor.js

```diff
@@ -94,6 +94,21 @@ class Anchor {
     const tx = this.contract.methods.anchorStateRoot(blockHeight, stateRoot);
     return Utils.sendTransaction(tx, txOptions);
   }
+
+  /**
+   * Get the latest state root and block height.
+   *
+   * @returns {Promise<Object>} Promise object that resolves to object containing state root and block height.
+   */
+  async getLatestAnchorInfo() {
```

Comment:
```suggestion
  async getLatestInfo() {
```
? 🤔

Reply:
Not sure if that isn't to vague? maybe `getLatestChainInfo`?

Reply:
I already made the suggested change, changed it to `getLatestInfo`.
I saw your comment now @hobofan. I too felt it is vague.